app/src/main/java/at/techbee/jtx/ui/detail/DetailsTopAppBar.kt

```diff
@@ -54,6 +56,21 @@ fun DetailsTopAppBar(
 ) {
 
     var textFieldText by rememberSaveable { mutableStateOf("") }
+    var previousText by rememberSaveable { mutableStateOf(emptyPreviousText) }
+
+    fun onSubtaskDone(value: String) {
+        val listOfSubtasks = value.split("\r\n", "\n", "\r")
```

Comment:
@t-h-e From my experience it's best to use System.lineSeparator() instead of "\r\n", "\n", "\r" =)

Reply:
If I would add a new line, I would also use `System.lineSeparator()`. But the code splits some text that it got from the user. We do not know where the user copied the text from. Therefore, the text could contain windows line endings or linux line endings, right?

Reply:
kotlin `.lineSequence()` also seems to splits by "\r\n", "\n", "\r"
Rather than splitting ourselves, we should probably use `.lines()`

Reply:
Sounds good, thanks :-)